rbc/tests/test_omnisci_udtf.py

```diff
@@ -169,7 +169,8 @@ def test_parallel_execution(omnisci, sleep, mode):
     (ct_sleep1).
 
     """
-    omnisci.require_version((5, 8), 'Requires omniscidb-internal PR 5901')
+    omnisci.require_version((5, 8), 'Requires omniscidb-internal PR 5901',
```

Comment:
https://github.com/omnisci/omniscidb-internal/pull/5901 has been merged. I don't know if the corresponding changeset is in 5.8 or 5.9, but if it is not, we should not use the label as the changeset is in master.

Reply:
Just tested and `ct_sleep` is not on omniscidb 5.8 but it is on 5.9. I'll update the PR accordingly.